### ISSUE TITLE
Accessibility:  NVDA/Jaws is not announcing the whole suggested information when we are typing in the copilot chat box

### ISSUE THREAD

**Issue by formulahendry** [microsoft/vscode#212322]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: 1.89.0
- OS Version: Windows 11 Enterprise 23H2 (OS build 22631.3296)

Copied from https://github.com/microsoft/vscode-azureapicenter/issues/102

### GitHub Tags
#A11yTCS; #A11ySev2; #Win32; #GH_VSCodeextensionforAPICenter_Win32_Apr2024; #DesktopApp; #VS Code Extension for API Center; #AILimited; #A11yMAS; #WCAG1.3.1; #Info and Relationships; #Jaws; #NVDA; #ScreenReader;

### Actual Result:
NVDA/Jaws is not announcing the whole suggested information when we are typing in the copilot chat box.

### Expected Result:
NVDA/Jaws should announce the whole suggested information when we are typing in the copilot chat box. for ex. if we are typing apicenter in the copilot chat then screen reader should also announce Build, consumer and consume great APIs.

### Environment Details:
Application Name: VS Code Extension for API Center
OS: Windows 11 Enterprise 23H2 (OS build 22631.3296)
NVDA Version: 2023.3.4

### Repro Steps:

1. Create an API center in azure portal.
2. Open Visual studio Code.
3. Tab till Chat option in the left side panel and press enter.
4. Type @apicenter chat box.
5. Verify whether the NVDA/Jaws is not announcing the whole suggested information when we are typing in the copilot chat box or not.

### User Impact:
Screen reader will not know about the whole information if the screen reader is not announcing it.

### Attachment:
**NVDA:**
![NVDA](https://github.com/microsoft/vscode-azureapicenter/assets/124244478/301b4732-0f56-4878-a02c-ab8a1864ca9b)
**JAWS:**
![Jaws](https://github.com/microsoft/vscode-azureapicenter/assets/124244478/5d28493b-a85c-4344-b39b-736d1913dd19)

**Comment by AnKushSingh05** [comment]
Primary Bug: https://github.com/microsoft/vscode-azureapicenter/issues/102

**Comment by meganrogge** [comment]
this is about the suggest widget descriptions not getting read @jrieken, I cannot find the issue for this but I think we've discussed it in the past?

**Comment by jrieken** [comment]
@meganrogge do you mean https://github.com/microsoft/vscode/issues/167900?

**Comment by formulahendry** [comment]
@AnKushSingh05 Please close original issue in API Center if it is duplicated.
